raiden/tests/utils/apitestcontext.py

```diff
@@ -34,13 +36,82 @@ def decode_response(response):
 class ApiTestContext():
 
     def __init__(self, reveal_timeout):
+        self.events = list()
         self.channels = []
         self.tokens = set()
         self.channel_schema = ChannelSchema()
         self.channel_list_schema = ChannelListSchema()
-        self.events_list_schema = EventsListSchema()
         self.reveal_timeout = reveal_timeout
 
+    def add_events(self, events):
+        self.events += events
+
+    def specify_token_for_channelnew(self, token_address):
```

Comment:
Question: Do we really this check? It feels like we are testing flask instead of our own code and it took me a bit to get what this state was being used for.